docs/how/auth/add-users.md

```diff
@@ -2,38 +2,43 @@
 
 Users can log into DataHub in 2 ways:
 
-1. Static credentials
-2. Single Sign-On via [OpenID Connect](https://www.google.com/search?q=openid+connect&oq=openid+connect&aqs=chrome.0.0i131i433i512j0i512l4j69i60l2j69i61.1468j0j7&sourceid=chrome&ie=UTF-8)
+1. Static credentials (Simplest)
+2. Single Sign-On via [OpenID Connect](https://www.google.com/search?q=openid+connect&oq=openid+connect&aqs=chrome.0.0i131i433i512j0i512l4j69i60l2j69i61.1468j0j7&sourceid=chrome&ie=UTF-8) (For Production Use)
 
-Option 1 is useful for running proof-of-concept exercises, while Option 2 is highly recommended for deploying DataHub in production.
+Option 1 is useful for running proof-of-concept exercises, or just getting DataHub up & running quickly. Option 2 is highly recommended for deploying DataHub in production.
 
 
 # Configuring static credentials
 
-## Step 1: Define a user.props file
+## Create a user.props file
 
-To define a set of username / password combinations that should be allowed to log in to DataHub, create a new file called `user.props`. This file should contain username:password combinations, with 1 user per line. For example, to create a `user.props` file with 2 users, the root
-"datahub" user and a custom user "johndoe", we would define the following file:
+To define a set of username / password combinations that should be allowed to log in to DataHub, create a new file called `user.props` at the file path `${HOME}/.datahub/plugins/auth/user.props`. 
+This file should contain username:password combinations, with 1 user per line. For example, to create 2 new users,
+with usernames "janesmith" and "johndoe", we would define the following file:
 
 ```
-# user.props
-datahub:rootpassword
+janesmith:janespassword
 johndoe:johnspassword
 ```
 
-We strongly recommend keeping a root user named `datahub` in your user.props. Otherwise, the root user will not be able to log in!
+Once you've saved the file, simply start the DataHub containers & navigate to `http://localhost:9002/login`
+to verify that your new credentials work.
 
-## Step 2: Mount user.props file to Docker container
+To change or remove existing login credentials, edit and save the `user.props` file. Then restart DataHub containers. 
 
-Once you've defined a `user.props` file, you'll need to mount the file into the `datahub-frontend` Docker container at the following path:
+If you want to customize the location of the `user.props` file, or if you're deploying DataHub via Helm, proceed to Step 2.
 
-```
-/datahub-frontend/conf/user.props
-```
+## (Advanced) Mount custom user.props file to container
+
+This step is only required when mounting custom credentials into a Kubernetes pod (e.g. Helm) **or** if you want to change
+the default filesystem location where DataHub checks for a custom `user.props` file (`${HOME}/.datahub/plugins/auth/user.props)`. 
```

Comment:
isn't this `/etc/datahub/plugins/auth/user.props` (as seen inside the container)

Reply:
No - this is if ${HOME}/.datahub/... doesn't work as a source location for you